### PR TITLE
PAYEE-1157: Amended discrepancy in print button

### PR DESCRIPTION
**Pull request by PerrettJ4**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by Jake-Raffe — APPROVED** [review]
lgtm

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala [hunk 1]
```diff
@@ -127,6 +127,7 @@ trait TaiViewSpec extends BaseSpec with JsoupMatchers {
   def pageWithContinueButtonFormNew(submitUrl: String): Unit =
     pageWithButtonFormNew(submitUrl, "Continue")
 
+//  TODO: This function is unused - Remove?
```
Comment (masahmad) [inline test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala:130]: do/ should  we replace the old button id with new id  enable test for new button

File: test/views/html/print/HistoricIncomeCalculationSpec.scala [hunk 2]
```diff
@@ -95,10 +95,6 @@ class HistoricIncomeCalculationSpec extends TaiViewSpec {
       doc must haveElementAtPathWithId("a", "backLink")
     }
 
-    "show the necessary print buttons" in {
-      doc.getElementsByClass("print-button").toString must include("javascript:window.print()")
-    }
-
```
Comment (masahmad) [inline test/views/html/print/HistoricIncomeCalculationSpec.scala:101]: did this test work previously -  can't see  "print-button" class on old button

Comment (PerrettJ4) [reply test/views/html/print/HistoricIncomeCalculationSpec.scala:101]: So it must have worked, but I couldn't get it work after refactoring (it's covered by the other test anyway)

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala (hunk 1 above)
Comment (PerrettJ4) [reply test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala:130]: As far as I'm aware we don't use this function though

File: test/views/html/print/HistoricIncomeCalculationSpec.scala (hunk 2 above)
Comment (PerrettJ4) [reply test/views/html/print/HistoricIncomeCalculationSpec.scala:101]: "print-button" class is in the _printWrapper.scala_
```
<div class="column-half print-button">
<a href="javascript:window.print();">@Messages("tai.label.print")</a>
</div>
```

Comment (PerrettJ4) [reply test/views/html/print/HistoricIncomeCalculationSpec.scala:101]: Should I try make it work?

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala (hunk 1 above)
Comment (PerrettJ4) [reply test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala:130]: and the new button doesn't have an id

File: test/views/html/print/HistoricIncomeCalculationSpec.scala (hunk 2 above)
Comment (PerrettJ4) [reply test/views/html/print/HistoricIncomeCalculationSpec.scala]: I made it work

File: test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala (hunk 1 above)
Comment (PerrettJ4) [reply test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala]: Deleted this function as not used

**Review by masahmad — APPROVED** [review]

**Pull request merged**